Sources/SwiftGRPCNIO/HTTP1ToRawGRPCServerCodec.swift

```diff
@@ -0,0 +1,122 @@
+import Foundation
+import NIO
+import NIOHTTP1
+
+public enum RawGRPCServerRequestPart {
+  case head(HTTPRequestHead)
+  case message(ByteBuffer)
+  case end
+}
+
+public enum RawGRPCServerResponsePart {
+  case headers(HTTPHeaders)
+  case message(ByteBuffer)
+  case status(GRPCStatus)
+}
+
+/// A simple channel handler that translates HTTP/1 data types into gRPC packets,
+/// and vice versa.
```

Comment:
Would be useful to explain why this is needed

Reply:
Done.